addons/test_website/static/tests/tours/website_settings.js

```diff
@@ -26,6 +26,7 @@ registry.category("web_tour.tours").add("website_settings_m2o_dirty", {
         {
             content: "open website switcher",
             trigger: "input[id^='website_id']",
+            run: "edit Test",
```

Comment:
Can be ignored if you pref but:
```suggestion
            run: `edit {websiteName}`,
```
or
```suggestion
            run: "edit Website Test",
```
Would imo make it more clear now that's it's explicit, I was wondering what was "Test"

Reply:
Done